### ISSUE TITLE
[Feature Request] Batch size scheduler to speed up training

### ISSUE THREAD

**Issue by felipheggaliza** [tensorflow/tensorflow#24179]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.12
- Are you willing to contribute it (Yes/No): Yes, but never contributed on TensorFlow code before.

**Describe the feature and the current behavior/state.**

I am looking for a batch size scheduler. Some recent papers like this one from Google (https://openreview.net/pdf?id=B1Yy1BxCZ) show that increasing batch size during training may have the same impact as decreasing the learning rate on validation accuracy. The advantage of increasing the batch size during training is that it requires less iterations per epoch, therefore reducing training time.

**Will this change the current api? How?**
Not sure.

**Who will benefit with this feature?**
Everyone in the TensorFlow community which are willing to train Neural Networks faster.

**Any Other info.**

**Reference paper:** *Don't Decay the Learning Rate, Increase the Batch Size 
Samuel L. Smith, Pieter-Jan Kindermans, Chris Ying, Quoc V. Le*
Link: https://openreview.net/pdf?id=B1Yy1BxCZ

**Comment by bhack** [comment]
I think that we can close this.

**Comment by rmothukuru** [comment]
@felipheggaliza,
Can you please let us know if the workaround mentioned in [this Comment](https://github.com/keras-team/keras/issues/10143#issuecomment-452144227) is what you are looking for? Thanks!
